salt/pillar/journal-cms.sls

```diff
@@ -31,6 +31,7 @@ journal_cms:
         articles_endpoint_for_migration: null
         articles_endpoint: null
         metrics_endpoint: null
+        digests_endpoint: null
```

Comment:
I'll take the existing `metrics_endpoint` as an example, which is quite similar.

`digests_endpoint` should be created in [these pillar files](https://github.com/elifesciences/builder-configuration/search?q=metrics_endpoint&unscoped_q=metrics_endpoint). With your permissions you should be able to open a PR there; after review and merge, this pillar will be immediately available to the various environments.